grammars/ruby.cson

```diff
@@ -214,7 +231,18 @@
     'name': 'variable.other.constant.ruby'
   }
   {
-    'begin': '(?x)\n\t\t\t         (?=def\\b)                                                      # an optimization to help Oniguruma fail fast\n\t\t\t         (?<=^|\\s)(def)\\s+                                              # the def keyword\n\t\t\t         ( (?>[a-zA-Z_]\\w*(?>\\.|::))?                                   # a method name prefix\n\t\t\t           (?>[a-zA-Z_]\\w*(?>[?!]|=(?!>))?                              # the method name\n\t\t\t           |===?|!=|>[>=]?|<=>|<[<=]?|[%&`/\\|]|\\*\\*?|=?~|[-+]@?|\\[]=?) )  # …or an operator method\n\t\t\t         \\s*(\\()                                                        # the openning parenthesis for arguments\n\t\t\t        '
+    'begin': '''(?x)
+      (?=def\\b)                          # optimization to help Oniguruma fail fast
+      (?<=^|\\s)(def)\\s+
+      (
+        (?>[a-zA-Z_]\\w*(?>\\.|::))?      # method prefix
+        (?>                               # method name
+          [a-zA-Z_]\\w*(?>[?!]|=(?!>))?
```

Comment:
Tried both ways, this way looked clearer. I'm not sure, r u referring to the format?